processor/otel/consumer.go

```diff
@@ -500,12 +503,28 @@ func translateSpan(span pdata.Span, metadata model.Metadata, event *model.Span)
 					netPeerName = stringval
 				}
 
-			// messaging
-			//
-			// TODO(axw) translate OpenTelemtry messaging conventions.
-			case "message_bus.destination":
+			// messaging.*
+			case "message_bus.destination", conventions.AttributeMessagingDestination:
 				message.QueueName = stringval
 				isMessagingSpan = true
+			case conventions.AttributeMessagingOperation:
+				if stringval == "" {
+					// TODO: Is this correct? Inferring that an empty string means "send", based on
+					// only `send`, `process`, and `receive` are allowed
+					// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md#operation-names
+					// `messaging.operation` can only have `process` and `receive` values
+					// cf. table in section https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md#apache-kafka-example
+					// and various examples
+					// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md#apache-kafka-example
+					stringval = "send"
+				}
+				messageOperation = stringval
```

Comment:
From reading the otel messaging spec about `message.operation`:

>The distinction between receiving and processing of messages is not always of particular interest or sometimes hidden away in a framework (see the Message consumption section above) and therefore **the attribute can be left out**.

(emphasis mine)

The span name also "should" be set, with the operation in the last position, but I would guess that has a higher likelihood of being respected. I'm wondering if it might be a good fallback to set the operation to this if `messageOperation=""` when setting `event.Action` below. Thoughts @axw ?

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md#span-name

Reply:
I don't have a strong opinion. I would be inclined to stay away from parsing, and:
 - use `message.operation` if it's set
 - use "send" if the span kind is PRODUCER

If you think it's important to also parse, I'm cool with that too.

Reply:
Sounds reasonable to me. I'd prefer not parsing as well.